modules/nw-sriov-configuring-device.adoc

```diff
@@ -55,18 +55,18 @@ spec:
 <4> Specify the node selector to select which node to be configured. User can choose to label the nodes manually or with tools like Kubernetes Node Feature Discovery.
 Only SR-IOV network devices on selected nodes will be configured. The SR-IOV
 CNI plug-in and device plug-in will be only deployed on selected nodes.
-<5> Specify an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`.
-<6> Specify a value for the maximum transmission unit (MTU) of the virtual function. The value for MTU must be in the range from `1` to `9000`. If you do not need to specify the MTU, specify a value of `''`. 
+<5> Optional. Specify an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`. The default value is `99`.
+<6> Optional. Specify a value for the maximum transmission unit (MTU) of the virtual function. The value for MTU must be in the range from `1` to `9000`.
```

Comment:
Is `9000` the max value admission controller rejects?
maybe be good to indicate that `The max mtu value may vary depend on different NIC models`

Reply:
`9000` limit is defined in the CRD.